schematic/store/synapse.py

```diff
@@ -1463,22 +1465,29 @@ def upload_manifest_file(
                 + file_extension
             )
 
-        manifestSynapseFile = File(
-            metadataManifestPath,
-            description="Manifest for dataset " + datasetId,
-            parent=datasetId,
-            name=file_name_new,
-        )
-        manifest_synapse_file_id = self.syn.store(
-            manifestSynapseFile, isRestricted=restrict_manifest
-        ).id
-
-        synapseutils.copy_functions.changeFileMetaData(
-            syn=self.syn,
-            entity=manifest_synapse_file_id,
-            downloadAs=file_name_new,
-            forceVersion=False,
-        )
+        try:
+            # Rename the file to file_name_new
+            original_file_path = metadataManifestPath
+            new_file_path = os.path.join(os.path.dirname(metadataManifestPath), file_name_new)
+            os.rename(original_file_path, new_file_path)
```

Comment:
To investigate: Do we need to write the file to disk with `metadataManifestPath`? Look at the code that is generating this and determine if we can instead just go ahead and write the CSV out to `file_name_new` instead.